receiver/http.go

```diff
@@ -47,11 +50,12 @@ least one handler to be set up, using Handle. This is done implicitly
 if the HTTP receiver is created using New()
 */
 type HTTP struct {
-	Address  string                        `doc:"Address to listen to." example:"[::1]:80 [2001:db8::1]:443"`
-	Handlers map[string]*skogul.HandlerRef `doc:"Paths to handlers. Need at least one." example:"{\"/\": \"someHandler\" }"`
-	Auth     map[string]*HTTPAuth          `doc:"A map corresponding to Handlers; specifying authentication for the given path, if required."`
-	Certfile string                        `doc:"Path to certificate file for TLS. If left blank, un-encrypted HTTP is used."`
-	Keyfile  string                        `doc:"Path to key file for TLS."`
+	Address       string                        `doc:"Address to listen to." example:"[::1]:80 [2001:db8::1]:443"`
+	Handlers      map[string]*skogul.HandlerRef `doc:"Paths to handlers. Need at least one." example:"{\"/\": \"someHandler\" }"`
+	Auth          map[string]*HTTPAuth          `doc:"A map corresponding to Handlers; specifying authentication for the given path, if required."`
+	Certfile      string                        `doc:"Path to certificate file for TLS. If left blank, un-encrypted HTTP is used."`
+	Keyfile       string                        `doc:"Path to key file for TLS."`
+	AcceptableCAs []string                      `doc:"Paths to files containing CAs which are accepted for Client Certificate authentication."`
```

Comment:
I'm not sure I like this naming. And I'm curious if we can also use the default system-wide CA? In that case, we could/would need two options. One to enable client certificate validation, the other to list custom CAs... That might be overkill though. Thoughts?

As for the name, regardless of if it's one or two: What about ClientCertificateCAs ? The reason I'm skeptical to AcceptableCAs is that client certificates is somewhat obscure to most people and as such just saying "AcceptableCAs" could mislead people to think it's about server-side certificates.